client/src/components/search/searchBar/SearchBar.test.js

```diff
@@ -0,0 +1,68 @@
+/* global jest, expect */
+import 'jest-dom/extend-expect';
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
This is great, however we are trying to move away from enzyme and use react-testing-library instead. You can find example usage in other tests.

Reply:
No problem. I'll take a look at some of the other tests and update this PR.